dev/Dockerfile.go-builder

```diff
@@ -1,2 +1,2 @@
-FROM golang:1.17.3
+FROM golang:1.18beta2
```

Comment:
Just bumping to `1.17.8` probably covers this. It's just linux vulns either way so if `1.17.3` image is updated that one will be fine too.

Reply:
concur